uktrade/data-hub-frontend
# Group similar export history items

## Description of change

This is a temporary FE fix to group the API data to reduce items in the list. The API is expected to do this in the future at which point this code will be removed.

It will group export history country changes to show the same actions on the same line. Currently if you add two countries they will show as two line items when it should be one.

## Test instructions

Edit the export countries either on the export tab or by adding an interaction with "Were countries discussed" as "Yes" and then click on "View full export countries history" and then countries added to the same category should appear in the same line

## Screenshots
### Before

![exports_before](https://user-images.githubusercontent.com/1481883/76093044-46858c80-5fb8-11ea-8fc3-f62d93ecb0b3.png)

### After

![export_after](https://user-images.githubusercontent.com/1481883/76093078-51402180-5fb8-11ea-86db-412f1420a983.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
